lib/standards/aria-roles.js

```diff
@@ -284,6 +303,10 @@ const ariaRoles = {
 		],
 		nameFromContent: true
 	},
+	meter: {
+		type: 'structure',
+		requiredAttrs: ['aria-valuemax', 'aria-valuemin', 'aria-valuenow']
```

Comment:
We'll need to add `aria-valuetext` in the `allowedAttrs` since we don't have inherited attrs.

Reply:
Okay, I think I got it fixed.